src/lib/Scenes/Sale/__tests__/helpers-tests.ts

```diff
@@ -6,10 +6,13 @@ const dAdd = (date: number, amount: number, type: "day" | "days") => moment(date
 const dSub = (date: number, amount: number, type: "day" | "days") => moment(date).subtract(amount, type).toISOString()
 
 describe(saleStatus, () => {
-  it("calculates correctly", () => {
+  it("returns the right sale status", () => {
     const now = Date.now()
-    expect(saleStatus(dAdd(now, 1, "day"), dAdd(now, 2, "days"))).toBe("notYetOpen")
-    expect(saleStatus(dSub(now, 1, "day"), dAdd(now, 2, "days"))).toBe("active")
-    expect(saleStatus(dSub(now, 3, "days"), dSub(now, 2, "days"))).toBe("closed")
+    expect(saleStatus(dAdd(now, 1, "day"), dAdd(now, 2, "days"), dAdd(now, 1, "day"))).toBe("notYetOpen")
+    expect(saleStatus(dAdd(now, 1, "day"), dAdd(now, 2, "days"), null)).toBe("notYetOpen")
+    expect(saleStatus(dSub(now, 1, "day"), dAdd(now, 2, "days"), dAdd(now, 1, "day"))).toBe("active")
+    expect(saleStatus(dSub(now, 1, "day"), dAdd(now, 2, "days"), null)).toBe("active")
+    expect(saleStatus(dSub(now, 3, "days"), dSub(now, 2, "days"), null)).toBe("closed")
+    expect(saleStatus(dSub(now, 3, "days"), dAdd(now, 2, "days"), dSub(now, 1, "day"))).toBe("closed")
```

Comment:
```suggestion
    expect(saleStatus(dSub(now, 3, "days"), dAdd(now, 2, "days"), dSub(now, 1, "day"))).toBe("closed")
    expect(saleStatus(dSub(now, 3, "days"), dSub(now, 2, "days"), null)).toBe("closed")
```
Just for OCD's sake :D